tests/Unit/Models/UserTest.php

```
@@ -151,7 +146,7 @@ public function it_can_get_matters_with_pending_tasks()
         $user = User::factory()->create(['login' => 'task.user']);
         $matter = Matter::factory()->create(['responsible' => 'task.user']);
         $event = Event::factory()->filing()->forMatter($matter)->create();
-        $task = Task::factory()->pending()->forEvent($event)->create(['code' => 'DL']);
+        $task = Task::factory()->pending()->forEvent($event)->create(['code' => 'REP']);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$task'. (undefined)

(UnusedLocalVariable)